versioneer.py

```diff
@@ -296,7 +296,7 @@ def get_root():
     versioneer_py = os.path.join(root, "versioneer.py")
     if not (os.path.exists(setup_py) or os.path.exists(versioneer_py)):
         # allow 'python path/to/setup.py COMMAND'
-        root = os.path.dirname(os.path.realpath(os.path.abspath(sys.argv[0])))
+        root = os.path.dirname(os.path.realpath(os.path.abspath(os.path.expanduser(sys.argv[0]))))
```

Comment:
This file is vendored and should not be edited (it's also part if the build/dev process and nothing normal users will ever see or interact with)